packages/presentation/lib/base/game_alert_dialog.dart

```diff
@@ -0,0 +1,8 @@
+abstract class DialogEvent{}
+
+class GameAlertDialog implements DialogEvent{
+  String titleText = 'Проверка 1';
```

Comment:
don't understand why do you use here default values, if you set a new data in constructor.

Reply:
left after performance check, deleted this text